### PR TITLE
698 add reference to list all templates

### PR DESCRIPTION
**Pull request by hamzarahal**

### CODE REVIEW

**Comment by bms63** [comment]
Hi @hamzarahal Hamza.  Just seeing where we are at with this PR?

**Review by amsmith214 — DISMISSED** [review]
think i've got this in good shape

**Review by amsmith214 — COMMENTED** [review]
on further review, need to try and bridge this PR and #702

**Comment by amsmith214** [comment]
kinda stumped as to why this is failing on release

**Comment by bms63** [comment]
> kinda stumped as to why this is failing on release

Stumped on why the test is failing?

**Comment by amsmith214** [comment]
> > kinda stumped as to why this is failing on release
> 
> Stumped on why the test is failing?

yeah. it passes 3.6, but fails on release, seemingly because of some differing column attributes, and i'm not sure how to explicitly control that, or where the difference is being introduced, especially as i'm using essentially the same code that passes everywhere else in the test files

**Comment by amsmith214** [comment]
thanks @bms63 !!!!

**Comment by wqrena** [comment]
@amsmith214 the unit test for max_dates looks good to me.

**Comment by thomas-neitmann** [comment]
This contains unrelated changes for `derive_vars_dtm()`. I would suggest to create a fresh PR that only include the updates to the documentation of the template functions.

**Comment by thomas-neitmann** [comment]
Please ignore my previous comment.

File: R/user_helpers.R [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #' Open an ADaM Template Script
 #'
+#' Uses list_all_templates() to discover which templates are available
```
Comment (thomas-neitmann) [inline R/user_helpers.R:3]: Please move this to the `@details` section. Also please put the function name into backticks. Plus it should not be Use~s~ but just Use.

**Review by thomas-neitmann — CHANGES_REQUESTED** [review]

File: R/user_helpers.R (hunk 1 above)
Comment (amsmith214) [reply R/user_helpers.R]: done

**Review by thomas-neitmann — APPROVED** [review]

**Pull request merged**